UnityClient/Assets/Scripts/TowerDefence/GameManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+
+    public static bool GameIsOver;
```

Comment:
`static` is strictly prohibited for use cases like this. It is brittle and not good design. Please redesign this to not be `static`. Thanks. Assuming this is public-facing data you might also want to consider this to be a property, or whatever fits your new design.